src/api/functions-api.js

```diff
@@ -23,6 +23,20 @@ export default {
 
     return mainHttpClient.get('/funcs', { params })
   },
+  getFunctionLogs: (project, name, tag, offset) => {
+    const params = {
+      project,
+      name,
+      tag,
+      logs: true
+    }
+
+    if (offset > 0) {
+      params.offset = offset
+    }
+
+    return mainHttpClient.get('/build/status?', { params })
```

Comment:
You don't need to add the `?` in the URL. Axios does that for you.

```suggestion
    return mainHttpClient.get('/build/status', { params })
```